tests/cypress/integration/stateWorkflow/stateSubmissionForm/documents.spec.ts

```diff
@@ -9,6 +9,17 @@ describe('documents', () => {
             const draftSubmissionID = pathname.split('/')[2]
             cy.visit(`/submissions/${draftSubmissionID}/documents`)
 
+            // Add two of the same document
```

Comment:
This is here until we can write a Jest test that covers this bug. So far, that has been elusive.